test/utils.cpp

```diff
@@ -814,3 +873,33 @@ TEST_CASE(
 				== "");
 	}
 }
+
+TEST_CASE(
+		"get_auth_method() returns properly"
```

Comment:
Please put a space at the end of the string literal if you want it to concatenate with the next one. C++ doesn't do it automatically, so in this case, we'll see "… returns properlyon defined …".

Reply:
Please expand this test to cover [this](https://coveralls.io/builds/20138514/source?filename=src%2Futils.cpp#L1120) line.